src/components/SingleBlog/index.jsx

```diff
@@ -0,0 +1,82 @@
+import React, { useState, useEffect } from "react";
+import "./index.scss";
+import { Container, Col, Row } from "react-bootstrap";
+import { useParams, Link } from "react-router-dom";
+import { ReactComponent as FacebookIcon } from "../../images/icons/facebookIcon.svg";
+import { ReactComponent as TwitterIcon } from "../../images/icons/twitterIcon.svg";
+import { ReactComponent as LinkedInIcon } from "../../images/icons/linkedInIcon.svg";
+import { ReactComponent as InstagramIcon } from "../../images/icons/instagramIcon.svg";
+import { useTranslation } from "react-i18next";
+
+const RELATED_POSTS_TO_SHOW = 4;
+
+export default function SingleBlog(props) {
+  const { t } = useTranslation();
+  const params = useParams();
+  const [blogPost, setBlogPost] = useState();
+
+  const fetchSingleBlogPost  = async () => {
+    const data = await fetch(
+      `https://public-api.wordpress.com/wp/v2/sites/safeplace102505649.wordpress.com/posts/${params.id}`
+    );
+    const posts = await data.json();
+    setBlogPost(posts);
+  };
+
+  useEffect(() => {
+    fetchSingleBlogPost ();
+  });
+
+  if (!blogPost) {
+    return <h1>loading</h1>;
```

Comment:
this needs translation too. and please change to h2

Reply:
if (!blogPost) {
  return loading
}

return (
)

you can save a level of indentation, you don t need the else